src/Sema.zig

```diff
@@ -9952,8 +9952,19 @@ fn zirBitcast(sema: *Sema, block: *Block, inst: Zir.Inst.Index) CompileError!Air
                 dest_ty.fmt(pt), container,
             });
         },
+        .array => {
+            const elem_ty = dest_ty.elemType2(zcu);
```

Comment:
please don't call this function it is my nemesis
```suggestion
            const elem_ty = dest_ty.childType(zcu);
```

Reply:
ah, indeed, suboptimal comment!

`childType` is exactly what you intuitively expect it to be. `elemType2` is a weird horrible function which does about fifty unrelated things and exactly one of them is arguably useful